bin/relate.py

```diff
@@ -194,7 +194,7 @@ def test_code_yml(yml_file, repo_root):
             return
 
         for page in pages:
```

Comment:
Not liking the rename much. This gets used as
```
relate validate .
```
or
```
relate test-code some/where.yml
```
Changing those to `relate.py` feels like it will hurt my muscle memory and add clutter for little gain.

That said, I am all for making this visible to the linter.

Reply:
Once you run `poetry install` it install the script with the name `relate`. So it doesn't change the end-user ergonomics at all, it just moves it closer to the code it depends on and makes it visible to linting/testing. 

E.g. this works as expected on my machine
```
poetry install
poetry shell
cd relate-sample
relate validate .
```
or
```
poetry install
cd relate-sample
poetry run relate validate .
```

You can even clone the repo right now and try it out. That part of it works perfectly on my machine.